### PR TITLE
Add DIP #60: ban fifth glyph

### PR DESCRIPTION
**Pull request by avafloww**

### CODE REVIEW

**Comment by NotNite** [comment]
image

**Comment by avafloww** [comment]
> image

It is crucial to distinguish that this glyph still, in this instant, subsists within Dalamud's community, including in that DIP mold of ours. This DIP plans to cut out that glyph.

A distinguishing factor of this PR is that it avoids this glyph in its own wording.

File: text/60-ban-fifth-glyph.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+[unresolved-questions]: #unresolved-questions
+
+- How will our approach find a solution to substitution of past information with that notorious fifth mark?
+- What aid should our community supply to individuals not truly familiar with this common wording, to assist with adapting to this approach?
+
+# Future possibilities
+
+[future-possibilities]: #future-possibilities
+
+Adopting this proposal could grow in a distinct Dalamud idiom, which may attract scholarly curiosity and linguistic analysis. Additionally, it might prompt similar laws in various modding associations, sparking a flood of innovation and imagination.
```
Comment (MKhayle) [inline text/60-ban-fifth-glyph.md:59]: what about xivlaunchr?
dropping 5th glyph?

Comment (avafloww) [reply text/60-ban-fifth-glyph.md:59]: ya, what if us humans just call it "XL"?

**Comment by philpax** [comment]
> > image
> 
> It is crucial to distinguish that this glyph still, in this instant, subsists within Dalamud's community, including in that DIP mold of ours. This DIP plans to cut out that glyph.
> 
> A distinguishing factor of this PR is that it avoids this glyph in its own wording.

image

**Comment by avafloww** [comment]
The E Advocacy Committee has generously offered me an all-expenses-paid holiday (not a bribe) to Japan, including an exclusive tour of Square Enix Headquarters, to reconsider my position on this issue. With that said, after much thinking, I revise my viewpoint and believe that E is essential to communication in the English language, and withdraw this DIP.

**Pull request closed without merging**